packages/prisma/schema.prisma

```diff
@@ -21,6 +26,7 @@ model User {
   password         String?
   source           String?
   signature        String?
+  roles            Role[]           @default([USER])
```

Comment:
Excellent 🤩

Reply:
I have a question, @catalinpit  @Mythie , why is it  `@default([USER])` with the brackets and not ` @default(USER)`

Reply:
@dephraiim Prisma schema expects array field to have array values. So, for the Role[] field, the default must be an array, not a scalar, afaik.

Reply:
For multiple roles, I understand